2-Pricing/src/components/Cart.tsx

```diff
@@ -13,39 +13,17 @@ const getSubscriptionByName = (
 
 interface CartProps {
   isMonthlyView: boolean;
-  cart: ICart;
   setIsCartOpen: Dispatch<SetStateAction<boolean>>;
-  setCart: Dispatch<SetStateAction<ICart>>;
 }
 
-export const Cart = ({
-  isMonthlyView,
-  cart,
-  setIsCartOpen,
-  setCart,
-}: CartProps) => {
+export const Cart = ({ isMonthlyView, setIsCartOpen }: CartProps) => {
+  const {
+    cart,
+    increment: handleIncrementCart,
+    decrement: handleDecrementCart,
+  } = useCartStore();
```

Comment:
In this case this doesn't matter because you are fetching everything from the store, but take a look at this for future reference:

https://stackoverflow.com/questions/68609189/fetching-multiple-states-with-zustand-react-shorthand-syntax